### PR TITLE
Update UI for TestKit Registration

### PR DESCRIPTION
**Pull request by sam-warren**
# Fixes or Implements [AB#12236](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/12236)

## Description

Create PCR test service and view.

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

-   [ ] Unit Tests Updated
-   [ ] Functional Tests Updated
-   [X] Not Required

### UI Changes

YES

manual
landing
nophn
authenticated

### Browsers Tested

-   [X] Chrome
-   [ ] Safari
-   [ ] Edge
-   [ ] Firefox

## Notes/Additional Comments

Test when 12238 is coplete.

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

-   Fulfills Work Item Requirements

    -   The changes meet/implement story's acceptance criteria. Fixes identified problems if bug.
    -   Changes not directly related to the task requirements need to be documented on the PR.

-   Compilation / Tests

    -   The changes do not break compilation and/or unit or functional tests; unless other item addresses them specifically.

-   Logic Problems / Functional Behaviour

    -   The changes work as intended and do not introduce problems.

-   Performance

    -   The changes do not introduce obvious performance issues.

-   Documented Standards

    -   The changes adhere to the team's documented [Coding Standards](https://github.com/bcgov/healthgateway/wiki/standards).

-   Readability / Maintainability
    -   The changes can be easily understood/read and allow for future enhancements without major refactoring. Readability is preferred over "clever code".
    -   Reasoning for changes needs to be clearly articulated. Disagreements should be arbitrated by a third developer.

### CODE REVIEW

**Comment by pagustinqt20** [comment]
@sam-warren  Can you add a screenshots (UI changes) please

**Comment by sam-warren** [comment]
> @sam-warren Can you add a screenshots (UI changes) please

Screenshots added

**Review by sslaws — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/components/navmenu/navHeader.vue [hunk 1]
```diff
@@ -103,6 +109,7 @@ export default class HeaderComponent extends Vue {
 
     private mounted() {
         this.loadOidcUserOnChange();
+        console.log("#### IS HEADER BUTTON SHOWN? " + this.isHeaderButtonShown);
```
Comment (BrianMaki) [inline Apps/WebClient/src/ClientApp/src/components/navmenu/navHeader.vue:112]: Nit: Was this console log intentional?

**Review by BrianMaki — APPROVED** [review]
Comment about console log message.

File: Apps/WebClient/src/ClientApp/src/components/navmenu/navHeader.vue (hunk 1 above)
Comment (sam-warren) [reply Apps/WebClient/src/ClientApp/src/components/navmenu/navHeader.vue:112]: Good catch! Will update

**Review by pagustinqt20 — APPROVED** [review]

**Pull request merged**